arkouda/pdarrayclass.py

```diff
@@ -1043,6 +1052,75 @@ def astype(self, dtype) -> pdarray:
 
         return akcast(self, dtype)
 
+    def slice_bits(self, low, high) -> pdarray:
+        """
+        Returns a pdarray containing only bits from low to high of self
+
+        Parameters
+        __________
+        low: int
+            The lowest bit included in the slice (inclusive)
+        high: int
+            The highest bit included in the slice (inclusive)
+
+        Returns
+        -------
+        pdarray
+            A new pdarray containing the bits of self from low to high
+
+        Raises
+        ------
+        RuntimeError
+            Raised if there is a server-side error thrown
+
+        Examples
+        --------
+        >>> p = ak.array([2**65 + (2**64 - 1)])
+        >>> bin(p[0])
+        '0b101111111111111111111111111111111111111111111111111111111111111111'
+
+        >>> bin(p.slice_bits(64, 65)[0])
+        '0b10'
+        """
+        if low > high:
+            raise ValueError("low must not exceed high")
+        return (self >> low) % 2**(high - low + 1)
+
+    def bigint_to_uint_arrays(self):
+        """
+        Creates a list of uint pdarrays from a bigint pdarray.
+        The first item in return will be the highest 64 bits of the
+        bigint pdarray and the last item will be the lowest 64 bits.
+
+        Returns
+        -------
+        List[pdarrays]
+            A list of uint pdarrays where:
+            The first item in return will be the highest 64 bits of the
+            bigint pdarray and the last item will be the lowest 64 bits.
+
+        Raises
+        ------
+        RuntimeError
+            Raised if there is a server-side error thrown
+
+        See Also
+        --------
+        pdarraycreation.bigint_from_uint_arrays
+
+        Examples
+        --------
+        >>> a = ak.arange(2**64, 2**64 + 5)
+        >>> a
+        array(["18446744073709551616" "18446744073709551617" "18446744073709551618"
+        "18446744073709551619" "18446744073709551620"])
+
+        >>> a.bigint_to_uint_arrays()
+        [array([1 1 1 1 1]), array([0 1 2 3 4])]
+        """
+        ret_list = json.loads(generic_msg(cmd="break_into_arrays", args={"array": self}))
```

Comment:
This is more of a nit-pick than anything, but I do think `break_into_arrays` may be a big vague. Maybe we could use something like `bigint_to_uint_list`.  Again there is nothing wrong with the current command, but it may become confusing for someone down the line who was not around during this development.